src/parser/rogue/shared/resources/ComboPointDetails.js

```diff
@@ -4,33 +4,68 @@ import Analyzer from 'parser/core/Analyzer';
 import Tab from 'interface/others/Tab';
 import StatisticBox, { STATISTIC_ORDER } from 'interface/others/StatisticBox';
 import Icon from 'common/Icon';
+import { formatPercentage } from 'common/format';
 import ResourceBreakdown from 'parser/shared/modules/resourcetracker/ResourceBreakdown';
 
 import ComboPointTracker from './ComboPointTracker';
 
+const MINOR_THRESHOLD = 0.05;
+const AVERAGE_THRESHOLD = 0.1;
+const MAJOR_THRESHOLD = 0.2;
 
 class ComboPointDetails extends Analyzer {
   static dependencies = {
     comboPointTracker: ComboPointTracker,
   };
 
+  get wasted() {
+    return this.comboPointTracker.wasted || 0;
+  }
+
+  get total() {
+    return this.comboPointTracker.wasted + this.comboPointTracker.generated || 0;
+  }
+
+  get wastedPercent() {
+    return this.wasted / this.total || 0;
+  }
+
+  get suggestionThresholdsWasted() {
```

Comment:
Is currently unused, a copy&paste error or are you planning on adding a suggestion for this?